spec/controllers/lists_controller_spec.rb

```diff
@@ -110,16 +111,25 @@
       it "creates a new list item and makes the correct calls to the Publishing API" do
         stub_user.update!(permissions: ["signin", "GDS Editor", "Redesigned lists"])
 
-        patch :update_list_items, params: { tag_id: tag.content_id, id: list.id, list: { list_items: ["/new-list"] } }
+        patch :update_list_items, params: {
+          tag_id: tag.content_id,
+          id: list.id,
+          list: { list_items: ["/new-list", "/newer-list"] },
+        }
 
-        new_list_item = list.reload.list_items.last
+        new_list_item1 = list.reload.list_items[-2]
+        new_list_item2 = list.list_items.last
```

Comment:
The `-2` here is a bit cryptic.

Assuming this is an active record collection I wonder if you can do: `new_list_item1, new_list_item2 = list.list_items.last(2)`

Reply:
Ooooh that's really cool. Thanks!